Dockerfile

```diff
@@ -70,10 +72,11 @@ RUN bundle config build.nokogiri --with-iconv-lib=/usr/local/lib --with-iconv-in
 
 COPY . /mastodon
```

Comment:
You can now [run chown while copying](https://docs.docker.com/engine/reference/builder/#copy), which is more efficient and saves you an additional run statement. I'd get rid of the `chown` statement in line 76 in favor of using `COPY --chown=${UID}:${GID}`?

Reply:
We could give it a try, indeed.